greek/use_rope/use_the_rope.md

```diff
@@ -0,0 +1 @@
+Αρπάζεις το σκοινί και αρχίζεις να τυλίγεις το ζαχαρωτό σε μια απεγνωσμένη προσπάθεια να το στραγγαλίσεις. Πάνω στην πάλη συνειδητοποιείς πως το ζαχαρωτό είναι πιο δυνατό από σένα και καταφέρνει να ξεφύγει από τα δεσμά σου, σπρώχνοντάς σε παράλληλα στην απέναντι γωνία του δωματίου.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.